tests/test_playbooks/tasks/registration_command.yml

```diff
@@ -0,0 +1,21 @@
+---
+- name: "Ensure registration command is {{ rc_state }}"
+  vars:
+    rc_organization: Test Organization
+    rc_location: Test Location
+    rc_state: present
+  registration_command:
+    username: "{{ foreman_username }}"
+    password: "{{ foreman_password }}"
+    server_url: "{{ foreman_server_url }}"
+    validate_certs: "{{ foreman_validate_certs }}"
+    organization: "{{ rc_organization }}"
+    location: "{{ rc_location }}"
+    state: "{{ rc_state }}"
```

Comment:
Is passing some `state` necessary for this module? What would it mean to ensure state for a registration command?

The documentation in the module has no state in the example:

```
EXAMPLES = '''
- name: "Generate registration command"
  theforeman.foreman.registration_command:
    username: "admin"
    password: "changeme"
    server_url: "https://foreman.example.com"
'''
```